<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Plotter" name="widgetDraw" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>661</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="pushButtonColor">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="title">
     <string>Cor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSlider" name="sliderR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>681</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSlider" name="sliderX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumberX"/>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelX">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumberZ"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelY">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelZ">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumberY"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxEllipsoid">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Ellipsoide</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QSlider" name="sliderRaioX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderRaioY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRy">
       <property name="text">
        <string>Ry</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderRaioZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelRx">
       <property name="text">
        <string>Rx</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelRz">
       <property name="text">
        <string>Rz</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxSphere">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Esfera</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSlider" name="sliderRaio">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberRaio">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRaio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Caixa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QSlider" name="sliderDimZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDy">
       <property name="text">
        <string>Dy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="sliderDimX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDz">
       <property name="text">
        <string>Dz</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderDimY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelDx">
       <property name="text">
        <string>Dx</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber_4"/>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumber_5"/>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_6"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSalvar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actioninsert_voxel"/>
   <addaction name="actionremove_voxel"/>
   <addaction name="actioninsert_box"/>
   <addaction name="actionremove_box"/>
   <addaction name="actioninsert_sphere"/>
   <addaction name="actionremove_sphere"/>
   <addaction name="actioninsert_ellipsoid"/>
   <addaction name="actionremove_ellipsoid"/>
   <addaction name="actionXY"/>
   <addaction name="actionXZ"/>
   <addaction name="actionYZ"/>
  </widget>
  <action name="actioninsert_voxel">
   <property name="text">
    <string>InserirVoxel</string>
   </property>
  </action>
  <action name="actionremove_voxel">
   <property name="text">
    <string>RemoverVoxel</string>
   </property>
  </action>
  <action name="actioninsert_box">
   <property name="text">
    <string>InserirCaixa</string>
   </property>
  </action>
  <action name="actionremove_box">
   <property name="text">
    <string>RemoverCaixa</string>
   </property>
  </action>
  <action name="actioninsert_sphere">
   <property name="text">
    <string>InserirEsfera</string>
   </property>
  </action>
  <action name="actionremove_sphere">
   <property name="text">
    <string>RemoverEsfera</string>
   </property>
  </action>
  <action name="actioninsert_ellipsoid">
   <property name="text">
    <string>InserirElipsoide</string>
   </property>
  </action>
  <action name="actionremove_ellipsoid">
   <property name="text">
    <string>RemoverEllipsoide</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionXY">
   <property name="text">
    <string>PlanoXY</string>
   </property>
  </action>
  <action name="actionXZ">
   <property name="text">
    <string>PlanoXZ</string>
   </property>
  </action>
  <action name="actionYZ">
   <property name="text">
    <string>PlanoYZ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
   <slots>
    <signal>updatePos(int,int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>widgetDraw</sender>
   <signal>updatePos(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>drawShape(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetDraw</sender>
   <signal>updatePos(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCoords(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pickColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>795</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelX</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelY</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelZ</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>672</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>633</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>636</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>674</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setR(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setG(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>633</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>642</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioninsert_voxel</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>insert_voxel_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionremove_voxel</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_voxel_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioninsert_box</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>insert_box_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioninsert_sphere</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>insert_sphere_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionremove_box</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_box_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionremove_sphere</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_sphere_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioninsert_ellipsoid</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>insert_ellipsoid_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionremove_ellipsoid</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_ellipsoid_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRaioX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>854</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRaioY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRaioZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>851</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRaio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRaio</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>819</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>852</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>856</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>799</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>861</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNovo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>new_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSalvar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionXY</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>surfaceXY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionXZ</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>surfaceXZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionYZ</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>surfaceYZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setR(int)</slot>
  <slot>setG(int)</slot>
  <slot>setB(int)</slot>
  <slot>pickColor()</slot>
  <slot>drawShape(int,int)</slot>
  <slot>updateCoords(int,int)</slot>
  <slot>insert_voxel_triggered()</slot>
  <slot>remove_voxel_triggered()</slot>
  <slot>insert_box_triggered()</slot>
  <slot>remove_box_triggered()</slot>
  <slot>insert_sphere_triggered()</slot>
  <slot>remove_sphere_triggered()</slot>
  <slot>insert_ellipsoid_triggered()</slot>
  <slot>remove_ellipsoid_triggered()</slot>
  <slot>new_triggered()</slot>
  <slot>save_triggered()</slot>
  <slot>surfaceXY()</slot>
  <slot>surfaceXZ()</slot>
  <slot>surfaceYZ()</slot>
 </slots>
</ui>
